renaud.mk

```diff
@@ -1,4 +1,4 @@
 INSTANCE_ID = renaud
 VARS_FILE = vars_renaud.yaml
-
+PRINT_OUTPUT = /tmp
```

Comment:
Shouldn't we create a new file like `luxambourg_demo.mk` (and `camptocamp_demo.mk` ?) where we can put the host specific things ?